features/ui/src/main/java/org/opennms/alec/rest/SituationRestImpl.java

```diff
@@ -292,7 +313,11 @@ private boolean alarmIsNotInAnotherSituation(String reductionKey) throws Interru
     private Response forwardAndStoreSituation(Situation oldSituation, Set<Alarm> alarms) {
         try {
             Situation newSituation = ImmutableSituation.newBuilderFrom(oldSituation).setAlarms(alarms).build();
+            //Forward situation to ONMS
             situationDatasource.forwardSituation(newSituation);
+            //Store  situation to the cloud
+            client.sendSituation(newSituation, runtimeInfo.getSystemId());
```

Comment:
Should revisit the implementation of `SituationClient` to use async rather than blocking subs. (If service is not reachable, operations should continue.)